ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DOWNLINK_IPV4_POOL_TESTS.yml

```diff
@@ -0,0 +1,48 @@
+default_node_types:
+  - node_type: spine
+    match_hostnames:
+      - .*spine\d
+  - node_type: l3leaf
+    match_hostnames:
+      - .*l3leaf\d
+
+spine:
+  defaults:
+    loopback_ipv4_pool: 192.168.0.0/24
+    bgp_as: 65000
+  nodes:
+    - name: downlink-ipv4-pool-spine1
+      downlink_pools:
+        - downlink_ipv4_pool: 10.0.1.0/24
+          downlink_interfaces: [Ethernet3, Ethernet4]
+        - downlink_ipv4_pool: 10.0.3.0/24
+          downlink_interfaces: [Ethernet5, Ethernet6]
+      id: 1
+    - name: downlink-ipv4-pool-spine2
+      downlink_pools:
+        - downlink_ipv4_pool: 10.0.2.0/24
+          downlink_interfaces: [Ethernet3, Ethernet4]
+        - downlink_ipv4_pool: 10.0.4.0/24
+          downlink_interfaces: [Ethernet5, Ethernet6]
```

Comment:
Please have some of the cases using the range syntax `[Ethernet5-6]` and also some where you have to index further into the list (ex for the first one use `Ethernet1-4`)